x/dex/keeper/grpc_query_order.go

```diff
@@ -34,26 +33,8 @@ func (k Keeper) GetOrders(c context.Context, req *types.QueryGetOrdersRequest) (
 	orders := k.GetOrdersByIds(ctx, req.ContractAddr, activeOrderIds.Ids)
 	response := &types.QueryGetOrdersResponse{Orders: []*types.Order{}}
 	for _, order := range orders {
-		k.decorateOrderStatus(ctx, &order)
+		order := order
```

Comment:
this is redundant, right?

Reply:
@udpatil not really this is another thing that linter was complaining about. See https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop for details